common/lib/common-utils/src/utils.ts

```diff
@@ -0,0 +1,15 @@
+/*!
```

Comment:
nit: could you rename this file `bufferShared.ts` or something similar?  `utils` seems too non-descriptive